src/components/Layout/Layout.tsx

```diff
@@ -147,8 +147,15 @@ export const Layout = ({
 
     pageHeadings.forEach((node) => {
       const { innerText, id, localName } = node as HTMLElement;
-      if (innerText && id && (localName == 'h2' || localName == 'h3')) {
+      if (innerText && id && localName == 'h2') {
         headings.push({
+          linkText: innerText,
+          hash: id,
+          level: localName,
+          subheadings: []
+        });
+      } else if (innerText && id && localName == 'h3') {
+        headings[headings.length - 1].subheadings.push({
```

Comment:
Oh, I missed this during my review. This us causing the console error check to fail. It's on pages that don't have a parent h2 because they have improper heading structure.

Reply:
Fixed the improper heading structure throughout.